docs/relationships.md

```diff
@@ -191,13 +190,13 @@ const firstDoc = docs[0]
 firstDoc.printingCompany.unset()
 ```
 
-### Create a new document with existing relations
+### Create a new file with existing relations
 
-Simply pass your relations as a third parameter to `create()`:
+Simply pass your reference as a third parameter to `create()`:
```

Comment:
This phrase would be clearer by adding a subject before `reference` are we talking about a file reference? a relationship reference? etc